metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/TaxonomyTree.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.persistence.entities;
+
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+
+@Entity
+@SequenceGenerator(allocationSize = 1, name = "TAXONOMY_TREE_SEQ", sequenceName = "taxonomy_tree_sequence")
+public class TaxonomyTree {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "TAXONOMY_TREE_SEQ")
+    private long id;
+
+    @ApiModelProperty(position = 2, dataType = "java.lang.String", notes = "Url to a taxonomySpecies")
+    @JoinColumn(name = "species_id", nullable = false, referencedColumnName = "taxonomyId", unique = true)
+    @OneToOne
+    private Taxonomy taxonomySpecies;
```

Comment:
What if we just call the attribute `species`? If we are in a taxonomy tree it should be obvious that the species is associated with a taxonomy, and that would also make unnecessary to set the property `@JoinColumn(name=...)`.

Applies to the rest of columns.

Reply:
the problem is with the naming of taxonomy class, we can't name it as class so included taxonomy to all variables.